coderd/workspaceagents.go

```diff
@@ -948,11 +948,11 @@ func (api *API) workspaceAgentReportStats(rw http.ResponseWriter, r *http.Reques
 		UserID:             workspace.OwnerID,
 		TemplateID:         workspace.TemplateID,
 		ConnectionsByProto: payload,
-		ConnectionCount:    int32(req.ConnectionCount),
-		RxPackets:          int32(req.RxPackets),
-		RxBytes:            int32(req.RxBytes),
-		TxPackets:          int32(req.TxPackets),
-		TxBytes:            int32(req.TxBytes),
```

Comment:
You don't want to lose this precision. One could easily exceed 4GB bytes transmitted.